src/test/java/com/google/api/codegen/testdata/billingtest_gapic.yaml

```diff
@@ -0,0 +1,30 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.billingtest.v1
```

Comment:
Maybe it might be best to not end the API name in "test", because we will have tests for it later, so we could end up with "BillingtestTest" or things like that. Maybe "Fakebilling" would be the best name.